src/components/__tests__/InfiniteHits.js

```diff
@@ -107,6 +128,21 @@ it('renders correctly with a custom item rendering', () => {
   expect(wrapper.html()).toMatchSnapshot();
 });
 
+it('renders correctly on the first page', () => {
+  __setState({
+    ...defaultState,
+    isFirstPage: true,
+  });
+
+  const wrapper = mount(InfiniteHits, {
+    propsData: {
+      showPrevious: true,
+    },
+  });
+
+  expect(wrapper.html()).toMatchSnapshot();
```

Comment:
We could add an assertion on the `button` to help understand what we actually test.